Aspire Budgeting/Facilities/AppCoordinator.swift

```diff
@@ -106,6 +114,18 @@ extension AppCoordinator {
     self.selectedFile = file
     fileValidator.validate(file: file, for: self.user!)
   }
+
+  func dashboardRefreshCallback() {
+    self.contentProvider
+      .getDashboard(for: self.user!,
+                    from: self.selectedFile!,
+                    using: self.dataLocationMap!) {
+                      self.dashboardVM =
+                        DashboardViewModel(result: $0,
+                                           refreshAction: self.dashboardRefreshCallback)
+                      self.objectWillChange.send()
+    }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/7aca4c0af853437035bd9c567cb0389c311a79de/Aspire Budgeting/Facilities/AppCoordinator.swift#L127"></span>Closure end should have the same indentation as the line that started it. Expected 6, got 4.<br />
<code>closure_end_indentation</code> <code>AppCoordinator.swift:127</code></td>
    </tr>
  </tbody>
</table>